### PR TITLE
HUB-1067: attach ingress and egress rule to lambda security group to talk to internal lb

### PR DESCRIPTION
**Pull request by ctazeez**

### CODE REVIEW

File: terraform/modules/hub/hub_key_rotation.tf [hunk 1]
```diff
@@ -116,6 +116,24 @@ resource "aws_security_group" "sg_hkr_lambda" {
   vpc_id = aws_vpc.hub.id
 }
 
+resource "aws_security_group_rule" "sg_hkr_lambda_ingress" {
```
Comment (simongurney1) [inline terraform/modules/hub/hub_key_rotation.tf:119]: I dont think ingress is required is it?

Comment (simongurney1) [reply terraform/modules/hub/hub_key_rotation.tf:119]: Ah is this ingress from the lambda?  If so, is it not just port 80?

Comment (ctazeez) [reply terraform/modules/hub/hub_key_rotation.tf:119]: added incase it required any other resource need to talk to lambda since its attached to vpc

Comment (ctazeez) [reply terraform/modules/hub/hub_key_rotation.tf:119]: its iingress for the lambda

Comment (simongurney1) [reply terraform/modules/hub/hub_key_rotation.tf:119]: ah, in that case I think it should be removed unless we actually need it

File: terraform/modules/hub/hub_key_rotation.tf [hunk 2]
```diff
@@ -116,6 +116,24 @@ resource "aws_security_group" "sg_hkr_lambda" {
...
+  type      = "ingress"
+  from_port = "0"
+  to_port   = "65535"
+  protocol  = "tcp"
+  cidr_blocks = [aws_vpc.hub.cidr_block]
+  security_group_id = aws_security_group.sg_hkr_lambda.id
+}
+
+resource "aws_security_group_rule" "sg_hkr_lambda_egress" {
```
Comment (simongurney1) [inline terraform/modules/hub/hub_key_rotation.tf:128]: I don't normally like open egress, but this is the behaviour of the existing lambdas so im fine with it

File: terraform/modules/hub/hub_key_rotation.tf (hunk 1 above)
Comment (ctazeez) [reply terraform/modules/hub/hub_key_rotation.tf]: Done

**Review by simongurney1 — APPROVED** [review]

**Pull request merged**